crown-court-proceeding/src/main/java/uk/gov/justice/laa/crime/crowncourt/client/RestAPIClient.java

```diff
@@ -107,4 +108,26 @@ public <R> R getGraphQLApiResponse(Class<R> responseClass,
                 .doOnError(Sentry::captureException)
                 .block();
     }
+
+    public ResponseEntity<Void> getApiResponseViaHEAD(String url,
```

Comment:
I think this can be replaced with `RestAPIClient` from `laa-crime-commons` see below

https://github.com/ministryofjustice/laa-crime-commons/blob/1404d22a99b17b095270cdb6d70369c918f0c715/crime-commons-spring-boot-starter-rest-client/src/main/java/uk/gov/justice/laa/crime/commons/client/RestAPIClient.java

Reply:
Agreed